h/oauth/jwt_grant.py

```diff
@@ -77,10 +77,9 @@ def create_token_response(self, request, token_handler):
         except errors.OAuth2Error as e:
             return headers, e.json, e.status_code
 
-        token = token_handler.create_token(
-            request, refresh_token=True, save_token=False
-        )
+        token = token_handler.create_token(request, refresh_token=True)
```

Comment:
This is actually 100% of the code to fix the upgrade for oauth. The rest is related to using templated URIs and adding test coverage.